coderd/rbac/scopes.go

```diff
@@ -8,39 +8,52 @@ import (
 
 type Scope string
 
+// TODO: @emyrk rename this struct
+type ScopeRole struct {
```

Comment:
Maybe this should be `Scope` and the string should be `ScopeID`?

Reply:
Still going to rename?

Reply:
I renamed `Scope` to `ScopeName`. So I can rename `ScopeRole` -> `Scope` again :+1: 

Will do now.